README.md

```diff
@@ -81,8 +81,9 @@ $ scripts/sign.sh PDF_FILE [SEED] [ALGO]
 ```
 Note that `SEED` is necessary when initializing the first hash, and the default `ALGO`  is sha256 if not specified manually.
 
-To verify if two pdf files are in the same chain, you can use the command:
+To verify if the given inputs are in the same chain, you can use the command:
 ```shell
-$ scripts/verify.sh PDF_QUERY PDF_ANCHOR [ALGO] [RANGE]
+$ scripts/verify.sh QUERY ANCHOR [ALGO] [RANGE]
```

Comment:
`QUERY` can be referred to `INPUT_PDF`.

Reply:
Alternatively, we may use 2 inputs, which can be specified later.

Reply:
`QUERY` is changed to `INPUT_PDF` now. Though the argument is specified to `*_PDF`, the script also accepts base64 encoded string.